History.md

```diff
@@ -1,3 +1,8 @@
+unreleased
+==========
+
+  * Ignore `Object.prototype` values in settings
```

Comment:
I can't remember, do we usually have these in the history in the PR? Seems like we did, but I would personally rather generate this when we build the release PR going forward. Not something we should block this on imo.

Reply:
Removed it.